scripts/tools/extract-tools-github.js

```diff
@@ -0,0 +1,22 @@
+const dotenv = require('dotenv');
+const axios = require('axios');
+dotenv.config();
```

Comment:
what do we need it for?

Reply:
Sorry, I added it before to get the environment variables right in the file but it didn't worked for me and I forgot to remove it during the commit. I'll remove this.